runtime/src/accounts_db.rs

```diff
@@ -148,11 +148,17 @@ pub struct AccountsAddRootTiming {
     pub store_us: u64,
 }
 
+#[derive(Debug, Default, Clone)]
+pub struct FillerAccountsConfig {
+    pub count: Option<usize>,
+    pub size: Option<usize>,
```

Comment:
1. Please remove `Option` from these fields.
2. Please add doc comments (`///`) for each field. Can be simple, like "number of filler accounts", and "data size per account, in bytes"